config/locales/en.yml

```diff
@@ -415,7 +415,8 @@ en:
           title_two: Is the trainee on either of these training initiatives?
           description: Training initiatives are sometimes referred to as programmes or schemes. They provide incentives and support for people training to teach. They’re separate from bursaries and scholarships.
         bursaries:
-          description_html: The course you have selected has a bursary available of %{amount} for %{subject}. You need to check if the trainee is eligible for this bursary </br></br> <a target="_blank" href="https://www.gov.uk/government/publications/initial-teacher-training-itt-bursary-fu[…]er-training-bursaries-funding-manual-2021-to-2022-academic-year">DfE bursary guidance for the academic year 2021 to 2022 (opens in new tab)</a>
+          description_html: The course you have selected has a bursary available of %{amount} for %{subject}. You need to check if the trainee is eligible for this bursary <br/><br/> <a target="_blank" href="https://www.gov.uk/government/publications/initial-teacher-training-itt-bursary-fu[…]er-training-bursaries-funding-manual-2021-to-2022-academic-year">DfE bursary guidance for the academic year 2021 to 2022 (opens in new tab)</a>
+          early_years_description_html: "%{training_route} has a bursary available of %{amount}. You need to check if the trainee is eligible for this bursary.<br/><br/> <a target=\"_blank\" href=\"https://www.gov.uk/government/publications/initial-teacher-training-itt-bursary-fu[…]er-training-bursaries-funding-manual-2021-to-2022-academic-year\">DfE bursary guidance for the academic year 2021 to 2022 (opens in new tab)</a>"
```

Comment:
There are some weird characters in the included anchor tag.
ive changed it https://github.com/DFE-Digital/register-trainee-teachers/pull/1101/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R431, but depending on when this goes it it might be worth fixing it here

Correct link should be https://www.gov.uk/government/publications/initial-teacher-training-itt-bursary-funding-manual/initial-teacher-training-bursaries-funding-manual-2021-to-2022-academic-year#training-bursary-award-and-eligibility